### ISSUE TITLE
Requirements: node 1.0?

### ISSUE THREAD

**Issue by forivall** [lcrossan/leaf#1]
By node 1.0, do you mean node 4.0 or iojs 1.0? I see you have destructring assignments. you need node 6 for that.

autumn... 🙄

**Comment by lcrossan** [comment]
yes i did much research and i can confirm thise code runs on nodeio version 0.10
